internal/testutil/fs.go

```diff
@@ -0,0 +1,31 @@
+package testutil
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+// tempDir supplements an alternative to TB.TempDir()[1], which is only available in 1.15.
+// [1]: https://github.com/golang/go/issues/35998
+func TempDir(t *testing.T) string {
+	dir, err := ioutil.TempDir("", t.Name())
```

Comment:
Let's make it run on Mac with the default configuration:

```suggestion
	tempDirRoot := "" // will use os.TempDir()
	if runtime.GOOS == "darwin" {
		// override the default since Docker for Mac
		// doesn't mount /var/folder by default where os.TempDir() will be located
		// See https://docs.docker.com/docker-for-mac/#file-sharing
		tempDirRoot = "/tmp"
	}

	dir, err := ioutil.TempDir(tempDirRoot, t.Name())
```